### ISSUE TITLE
subtract text mesh from geometry mesh 

### ISSUE THREAD

**Issue by mrballer** [chandlerprall/ThreeCSG#47]
hello,

is there an example available to subtract a text mesh from another geometry mesh?
I tried to play around with text meshes and geometry meshes. is it even possible?

best regards

**Comment by superodi** [comment]
Yes, it is. But you can look into the mesh the letters were substracted from so it seems to be not as usefull as wished.
Would be nice if developer would fix it.
